Add installation instructions (#134)
echemdb/svgdigitizer: doc/installation.md

Installation
============

The recommended way to install the svgdigitizer is to use your package manager,
(e.g., `apt-get` on Debian or Ubuntu, `pacman` on Arch Linux, `brew` on macOS.)

> You can consult [repology](https://repology.org/project/svgdigitizer/packages)

Author: Currently, that's of course not terribly helpful. Let's see how quickly this project is being picked up by the distributions.

Reviewer: Can we postpone this to a later moment?

Author: We're actually listed now:

```suggestion
You can consult [repology](https://repology.org/project/python:svgdigitizer/versions)
```

to see if the svgdigitizer is available for your package manager.

> Alternatively, the svgdigitizer can easily be installed by one of the following

Reviewer: ```suggestion
Alternatively, the svgdigitizer can be installed by one of the following
```

Reviewer: We tend to avoid words such as "easily" in our publications since this always depends on the users' skills.

methods.

Install with pip from PyPI
--------------------------

The latest stable version of the svgdigitizer is available on
[PyPI](https://pypi.org/project/svgdigitizer/) for all platforms and can be

> easily if you have Python and pip installed already:

Author: ```suggestion
installed if you have Python and pip installed already:
```

```sh
pip install svgdigitizer
```

This command downloads and installs the svgdigitizer and its dependencies into
your local Python installation.

> If the above command fails because you don't have permission to modify your

Author: ```suggestion
If the above command fails because you do not have permission to modify your
```

Python installation, you can install the svgdigitizer into your user account:

```sh
pip install --user svgdigitizer
```

> You can instead also install the latest unreleased version the svgdigitizer

Author: ```suggestion
You can instead also install the latest unreleased version of the svgdigitizer
```

from our [GitHub Repository](https://github.com/echemdb/svgdigitizer) with

```sh
pip install git+https://github.com/echemdb/svgdigitizer@master
```

Install with conda from conda-forge
-----------------------------------

The svgdigitizer is [available on
conda-forge](https://github.com/conda-forge/svgdigitizer-feedstock) for all
platforms.

If you don't have conda yet, we recommend to install
[Miniforge](https://github.com/conda-forge/miniforge#miniforge3).

Miniforge is already pre-configured for conda-forge. If you already had another

> release of conda installed, make sure that conda-forge channel is

Author: ```suggestion
release of conda installed, make sure the conda-forge channel is
```

[configured correctly](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge)

Once your conda setup is ready, create a new `svgdigitizer` environment with

> the latest stable version of the svgdigitizer

Author: ```suggestion
the latest stable version of the svgdigitizer:
```

```sh
conda create -n svgdigitizer svgdigitizer
```

To use the svgdigitizer, activate the `svgdigitizer` environment:

```sh
conda activate svgdigitizer
svgdigitizer --help
```

To install the svgdigitizer into an existing environment, activate that environment and then

```sh
conda install svgdigitizer
```

Install with pip for development
--------------------------------

If you want to work on the svgdigitizer itself, get a copy of the latest

> unreleased version the svgdigitizer:

Author: ```suggestion
unreleased version of the svgdigitizer:
```
